pmd-jsp/src/main/java/net/sourceforge/pmd/lang/jsp/rule/JspRuleViolationFactory.java

```diff
@@ -25,6 +28,16 @@ protected RuleViolation createRuleViolation(Rule rule, RuleContext ruleContext,
         return new ParametricRuleViolation<>(rule, ruleContext, (JspNode) node, message);
     }
 
+    @Override
+    protected AutoFixableRuleViolation createRuleViolation(Rule rule, RuleContext ruleContext, Node node, String message, Class<? extends RuleViolationFix> ruleViolationFixClass) {
+        return new AutoFixableParametricRuleViolation<>(rule, ruleContext, (JspNode) node, message, ruleViolationFixClass);
+    }
+
+    @Override
+    protected AutoFixableRuleViolation createRuleViolation(Rule rule, RuleContext ruleContext, Node node, String message, int beginLine, int endLine, Class<? extends RuleViolationFix> ruleViolationFixClass) {
+        return null;
```

Comment:
this should be implementable the same way you did for other languages such as PLSQL